### PR TITLE
Change map tile

### PR DESCRIPTION
**Pull request by cesswairimu**
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt jasmine`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [ ] PR is descriptively titled
* [ ] PR body includes `fixes #0000`-style reference to original issue #
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by cesswairimu** [comment]
The map will look as 
Screenshot from 2020-01-02 00-57-01
Once https://github.com/publiclab/leaflet-blurred-location/pull/228 is published

**Comment by cesswairimu** [comment]
I want to achieve this https://github.com/publiclab/plots2/pull/6817#issuecomment-566225019 for the simple editor template...what do you guys think about changing the tile to the bluish all together for the map module?

**Comment by nstjean** [comment]
I like that!!

File: src/modules/PublicLab.MapModule.js [hunk 1]
```diff
@@ -23,13 +23,17 @@ module.exports = PublicLab.MapModule = PublicLab.Module.extend({
          lngId: 'lng'
        }
      }
+     var token = "pk.eyJ1Ijoianl3YXJyZW4iLCJhIjoiVzVZcGg3NCJ9.BJ6ArUPuTs1JT9Ssu3K8ig";
+
+     options.tileLayerUrl = 'https://api.mapbox.com/styles/v1/mapbox/outdoors-v11/tiles/{z}/{x}/{y}?access_token=' + token;
```
Comment (sagarpreet-chadha) [inline src/modules/PublicLab.MapModule.js]: Hey i like this tileLayer :)
But i think we should also accept this as an argument when we initialize the editor object to overwrite this. What do you think? Thanks!

Comment (cesswairimu) [reply src/modules/PublicLab.MapModule.js]: Sounds good...Adding that. Thanks

**Comment by cesswairimu** [comment]
Hey @sagarpreet-chadha, I just added that, also which tile should we set as default? Thanks

**Comment by jywarren** [comment]
Shall we make the colorful ones default?

**Comment by cesswairimu** [comment]
Cool... That is what I have as the default here

**Comment by jywarren** [comment]
Great - just ping me when this is good to go! Thank you!

**Comment by jywarren** [comment]
Actually you can go ahead and merge when you're ready, Cess!

**Comment by cesswairimu** [comment]
@jywarren Its ready..please merge...I have no merge rights on this repo ..plus feels weird to merge my own PR :smile:

**Comment by jywarren** [comment]
lol ok! Sorry! And I can give you merge rights here if you'd like?

**Comment by cesswairimu** [comment]
No need, its fine. Thanks

**Comment by cesswairimu** [comment]
@nstjean @jywarren, do we need to bump up the version so that the changes can reflect on plots2?

**Comment by jywarren** [comment]
Yes, that's right!

**Comment by cesswairimu** [comment]
cool, opening a PR for that

**Pull request merged**